### PR TITLE
prime number code added

### PR DESCRIPTION
**Pull request by govindjha2017**
please review my prime number code

### CODE REVIEW

File: prime [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include<iosteam>
```
Comment (jhaprabhakar1998) [inline prime]: @JHAGOVIND Have you tried running the code. It is missing .cpp extension. Before creating a PR please ensure that coding is running fine

File: prime [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+using namespace std;
+int main{
+    int n;
+    cout<<"enter a number"<<endl;
```
Comment (jhaprabhakar1998) [inline prime]: No need to add cout statement as this code will be used as a helper code. So no print statements apart form input/output

Comment (jhaprabhakar1998) [reply prime:5]: Pleaser resolve above comments

**Comment by jhaprabhakar1998** [comment]
Resolve comments

**Review by jhaprabhakar1998 — COMMENTED** [review]
Don't push a.exe files. Delete it

File: prime.cpp [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#include<iostream>
+using namespace std;
+bool prime(int n){
+    if(n==1){
+        return false;
+    }
+    if(n==2){
+        return true;
+    }
```
Comment (jhaprabhakar1998) [inline prime.cpp]: Can we combine both if loop in one if condition @JHAGOVIND

File: prime.cpp [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    int temp=0;
```
Comment (jhaprabhakar1998) [inline prime.cpp]: Can we do it without using temp variable.

Comment (govindjha2017) [reply prime.cpp]: yes i can do

File: prime.cpp [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#include<iostream>
+using namespace std;
+bool prime(int n){
+    if(n<=1){
+        return false;
+    }
+    
+    for(int i=2;i<n;i++){
+        if(n%i==0){
```
Comment (jhaprabhakar1998) [inline prime.cpp]: It's complexity is O(n). We can do a lot better here

**Comment by govindjha2017** [comment]
now in my code time complexity is order of squire root of n

**Review by jhaprabhakar1998 — DISMISSED** [review]

**Review by Jhajitu200 — COMMENTED** [review]
change you file nmame

**Review by Jhajitu200 — APPROVED** [review]

**Review by Jhajitu200 — APPROVED** [review]

**Review by Jhajitu200 — APPROVED** [review]

**Review by jhaprabhakar1998 — APPROVED** [review]

**Review by Jhajitu200 — COMMENTED** [review]
correct

**Pull request merged**